geronimo-json_1.0_spec/src/main/java/javax/json/spi/JsonProvider.java

```diff
@@ -28,6 +28,9 @@
 import javax.json.stream.JsonGeneratorFactory;
 import javax.json.stream.JsonParser;
 import javax.json.stream.JsonParserFactory;
+
+import org.apache.geronimo.osgi.locator.ProviderLocator;
```

Comment:
the required classes are embedded in the jar file (via maven-bundle-plugin) - so the resulting jar file does no longer have a dependency to `geronimo-osgi-locator` - neither in OSGi nor in maven.
it's the same pattern as used by geronimo-jaxrs_1.1_spec for example.